include/Yash.h

```diff
@@ -394,11 +393,11 @@ class Yash {
     CommandSpan m_commands;
     std::array<std::string_view, TConfig.maxRequiredArgs> m_commandArgs;
     std::function<void(const char*)> m_printFunction;
-    std::vector<std::string> m_commandHistory;
-    std::vector<std::string>::const_iterator m_commandHistoryIndex;
-    std::string m_command;
-    std::string m_prompt;
-    std::string m_ctrlCharacter;
+    std::list<std::string> m_commandHistory;
+    std::list<std::string>::const_iterator m_commandHistoryIndex;
+    std::string m_command { "" };
+    std::string m_prompt { "Yash$ " };
```

Comment:
"Yash$ " why this change?

Reply:
Empty prompt looks weird and I thought that it was a nice gimmick :)